src/regtech_data_validator/data_formatters.py

```diff
@@ -156,10 +156,13 @@ def df_to_dicts(df: pd.DataFrame, max_records: int = 10000, max_group_size: int
                 total_errors_per_group[group_name] = max_group_size
         for validation_id, group in df.groupby("validation_id"):
             check = find_check(validation_id, checks)
-            truncated_group = truncate_validation_group_records(group, total_errors_per_group[validation_id])
+            truncated_group, need_to_truncate = truncate_validation_group_records(
+                group, total_errors_per_group[validation_id]
+            )
             group_json = process_chunk(truncated_group, validation_id, check)
             if group_json:
-                json_results.append(process_chunk(truncated_group, validation_id, check))
+                group_json["validation"]["truncated"] = need_to_truncate
```

Comment:
I think @billhimmelsbach mentioned `is_truncated` was the attribute name they wanted?